tests/network/vmi_networking.go

```diff
@@ -525,8 +525,13 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			dnsVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskCirros), userData)
 
 			dnsVMI.Spec.DNSPolicy = "None"
+
+			// This IPv4 address tests backwards compatibility of the "DNSConfig.Nameservers" field.
+			// The leading zero is intentional.
+			// For more details please see: https://github.com/kubevirt/kubevirt/issues/6498
+			const DNSNameserverWithLeadingZeros = "01.1.1.1"
```

Comment:
I think this will pass even on 1.17, as it `01` is `1` and in the right range. Also, consider using a loopback address so no surprises will occur. I do not know why the current addresses used are public ones.

Reply:
Without this PR, on go 1.17 - this address will fail on the validating webhook with the following error:
```
$ ./cluster-up/kubectl.sh apply -f examples/vmi-fedora.yaml 
...
The request is invalid: 
* nameservers: must be valid IP address: 01.1.1.1
```

With this PR, on go 1.17 - this address will be accepted, but will fail when reading /etc/resolv.conf from virt-launcher's pod (for using it as an input for the DHCP server).

As for your second comment, the addresses here don't really matter because we only test that the DHCP server had sent the VM a proper address, and then the VM is terminated.

Reply:
BTW, 1.1.1.1 is Cloudflare's DNS server